content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter/01-global-cutome-rroles.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Palette Resource Roles"
+metaTitle: "Palette Global and Custom Resource Roles "
+metaDescription: "Palette Global and Custom Resource Roles "
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+
+# Overview
+
+The page encompasses the Palette Resource Roles. We support two broad classifications of Resource Roles:
+
+<br />
+
+* [Palette Global Resource Roles](/clusters/cluster-management/cluster-tag-filter/global-cutome-rroles#globalresourceroles), the set of roles built in and available to you.
+
+
+* [Palette Custom Roles](/clusters/cluster-management/cluster-tag-filter/global-cutome-rroles#customroles), the roles you can create in Palette using our available set of permissions and operations.
+   * [Create Custom Role](/clusters/cluster-management/cluster-tag-filter/global-cutome-rroles#createcustomroles), how to create a custom role in the Palette console.
+
+
+## Global Resource Roles 
+
+The set of Global Resource Roles available within the Palette are:
+
+<br />
+
+* [Resource Cluster Admin](/user-management/palette-rbac/project-scope-roles-permissions#clusteradmin)
+
+
+* [Resource Cluster Editor](/user-management/palette-rbac/project-scope-roles-permissions#clustereditor)
+
+
+* [Resource Cluster Profile Admin](/user-management/palette-rbac/project-scope-roles-permissions#clusterprofileadmin)
+
+
+* [Resource Cluster Profile Editor](/user-management/palette-rbac/project-scope-roles-permissions#clusterprofileeditor)
+
+
+* [Resource Cluster Profile Viewer](/user-management/palette-rbac/project-scope-roles-permissions#clusterprofileviewer)
+
+
+* [Resource Cluster Viewer](/user-management/palette-rbac/project-scope-roles-permissions#clusterviewer)
+
+
+
+##  Custom Roles
+
+
+
+You can [create your set of custom resource roles]() in the Palette console and attach them to the user. The set of customizable Permissions and Operations supported by Palette to create custom roles are as follows:
+
+<br />
+
+1. CloudAccount
```

Comment:
Resma, each one of these should be its own table with the following column (Action | Description). The user needs to understand what each permission does. It might seem intuitive but we need to define the action.